src/foundation.cs

```diff
@@ -6777,11 +6777,17 @@ interface NSStream {
 		[Protocolize]
 		NSStreamDelegate Delegate { get; set; }
 
-		[Export ("propertyForKey:"), Internal]
-		NSObject PropertyForKey (NSString key);
+#if XAMCORE_4_0
+		[Abstract]
```

Comment:
But this is on the NSStream no in the NSInputStream, which when we change the API will have the method, correct?